src/neo/SmartContract/Native/Oracle/OracleRequest.cs

```diff
@@ -0,0 +1,41 @@
+using Neo.IO;
+using Neo.VM;
+using Neo.VM.Types;
+using Array = Neo.VM.Types.Array;
+
+namespace Neo.SmartContract.Native.Oracle
+{
+    public class OracleRequest : IInteroperable
+    {
+        public UInt256 Txid;
+        public string Url;
+        public string Filter;
+        public UInt160 CallbackContract;
+        public string CallbackMethod;
+        public byte[] UserData;
+
+        public void FromStackItem(StackItem stackItem)
+        {
+            Array array = (Array)stackItem;
+            Txid = new UInt256(array[0].GetSpan());
+            Url = array[1].GetString();
+            Filter = array[2].GetString();
+            CallbackContract = new UInt160(array[3].GetSpan());
+            CallbackMethod = array[4].GetString();
+            UserData = array[5].GetSpan().ToArray();
+        }
+
+        public StackItem ToStackItem(ReferenceCounter referenceCounter)
+        {
+            return new Array(referenceCounter)
+            {
+                Txid.ToArray(),
+                Url,
+                Filter ?? StackItem.Null,
```

Comment:
If it's `null`, It's already converted to null, isn't it?

Reply:
I'm afraid not.

https://github.com/neo-project/neo-vm/blob/baf4989f75353cc1b1383bdde916331b44843d89/src/neo-vm/Types/StackItem.cs#L125-L128

https://github.com/neo-project/neo-vm/blob/baf4989f75353cc1b1383bdde916331b44843d89/src/neo-vm/Types/ByteString.cs#L76-L79